### PR TITLE
Workshop ID 5921: GoldenGate Stream Analytics - Your HelloWorld Pipeline in 15 minutes!

### PR DESCRIPTION
**Pull request by prmanjun**
Formatting and date changes.

### CODE REVIEW

File: data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+- Create a Pipeline
+- Add a Query Stage to the source stream
+- Add a Filter to the Query stage
+- Publish the Pipeline
+
+### Prerequisites
+This Lab assumes you have:
+- A Free Tier, Paid, or LiveLabs Oracle Cloud account
+- SSH Private Key to access the host via SSH
+- An On-prem or a Marketplace instance of GoldenGate Stream Analytics (GGSA) version 19.1:
```
Comment (rfontcha) [inline data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md:27]: Remove this line

File: data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
```
Comment (rfontcha) [inline data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md:28]: @prmanjun - Move everything from this line to the "Learn more" section

**Review by rfontcha — CHANGES_REQUESTED** [review]
@prmanjun - Please see my comments in-line "1-ggsa-hwp"

**Review by klazarz — CHANGES_REQUESTED** [review]
@prmanjun please also use desriptions for your images, see self QA doc

File: data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md (hunk 1 above)
Comment (prmanjun) [reply data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md]: Closed

File: data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md (hunk 2 above)
Comment (prmanjun) [reply data-management-library/goldengate/stream-analytics-helloworld-pipeline/1-ggsa-hwp/1-ggsa-hwp.md]: Closed

**Comment by prmanjun** [comment]
> @prmanjun please also use desriptions for your images, see self QA doc



> @prmanjun please also use desriptions for your images, see self QA doc

Closed

**Review by klazarz — APPROVED** [review]

**Review by rfontcha — APPROVED** [review]

**Pull request merged**